addons/transformations.md

```diff
@@ -41,13 +41,7 @@ Transformation files need to be placed in the directory `$OPENHAB_CONF/transform
 
 2.  Rules
 
-    Transformations can also be used in Rules to **transform/translate/convert data**.
-    The general syntax is as follows:
-
-    ```java
-    transform("<transformation-identifier>", "<transf. expression or transf. file name>", <input-data or variable>)
-    ```
-
+    Transformations can also be [used in rules]({{base}}/configuration/rules-dsl.html#transformations) to **transform/translate/convert data**.
```

Comment:
@ThomDietrich I am not sure what you intend to convey with the bolded text.  Not a big deal, but personally I would prefer regular text.  Or maybe there is a reason it needs to be bold, but in that case, you might want to look at the section to see if there is a better way to convey the importance of what you are highlighting.

Reply:
No reason. Will remove it. 👍